packages/workbox-build/src/_types.js

```diff
@@ -26,3 +26,81 @@ import './_version.mjs';
  *
  * @memberof module:workbox-build
  */
+
+/**
+ * @typedef {Object} RuntimeCachingEntry
+ *
+ * @property {string|workbox.routing.Route~handlerCallback} handler
```

Comment:
I'm going to merge https://github.com/GoogleChrome/workbox/pull/2312 in a bit, and once that's done these should reference the new `module:workbox-routing`-style identifiers (here and throughout).

Reply:
Okay, I can update this PR once that's submitted.